### PR TITLE
Switch to our own LZO wrapper

### PR DESCRIPTION
**Pull request by gniezen**
See attached Trello card for details.

### CODE REVIEW

**Comment by gniezen** [comment]
[Find/use non-GPL LZO library for uploader](https://trello.com/c/IMEQd5Pm/491-find-use-non-gpl-lzo-library-for-uploader)

File: lib/drivers/medtronic600/medtronic600Driver.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const { promisify } = require('util');
 const ExtendableError = require('es6-error');
 const uploadDataPeriod = require('../../../app/utils/uploadDataPeriod');
 const crcCalculator = require('../../crc.js');
-const lzo = require('../../lzo');
+const lzo = require('lzo-decompress');
```
[the remark this answers is not in the document]
Comment (pazaan) [reply lib/drivers/medtronic600/medtronic600Driver.js]: I reviewed and tested it, but you're right in that it wasn't formally reviewed in a PR.

[the remark this answers is not in the document]
Comment (gniezen) [reply lib/drivers/medtronic600/medtronic600Driver.js]: I thought we documented that in the Trello card, but I can't find anything there. @pazaan , can you please review the latest `lzo-decompress` code and comment on the card to that extent? From here on we will be using regular PRs for code review on that repo.

[the remark this answers is not in the document]
Comment (pazaan) [reply lib/drivers/medtronic600/medtronic600Driver.js]: We did the review in Slack. I'll put a comment on the Trello card.

[the remark this answers is not in the document]
Comment (gniezen) [reply lib/drivers/medtronic600/medtronic600Driver.js]: Linking to the commit that contained my changes to your review was a great idea, thanks!

**Pull request merged**